gradient_z_min: -0.2
gradient_z_max: 0.05
gradient_dif_min : 0.03
gradient_dif_max : 0.05
rail_support_feature_thresh : 5
rail_support_feature_section_thresh : 0.1
